custom-login/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.2.0",
   "private": true,
   "dependencies": {
-    "@okta/okta-react": "^3.0.8",
+    "@okta/okta-auth-js": "^4.1.0",
+    "@okta/okta-react": "file:.yalc/@okta/okta-react",
```

Comment:
update after 4.0 release

Reply:
Can we keep this PR as Draft until this is updated?